notebooks/official/automl/sdk_automl_tabular_forecasting_batch.ipynb

```diff
@@ -505,20 +473,27 @@
       "source": [
         "#### Run the training pipeline\n",
         "\n",
-        "Next, you start the training job by invoking the method `run`, with the following parameters:\n",
+        "Next, start the training job by invoking the `run` method, with the following parameters:\n",
         "\n",
-        "- `dataset`: The `Dataset` resource to train the model.\n",
-        "- `model_display_name`: The human readable name for the trained model.\n",
-        "- `training_fraction_split`: The percentage of the dataset to use for training.\n",
-        "- `test_fraction_split`: The percentage of the dataset to use for test (holdout data).\n",
-        "- `target_column`: The name of the column to train as the label.\n",
-        "- `budget_milli_node_hours`: (optional) Maximum training time specified in unit of millihours (1000 = hour).\n",
+        "- `dataset`: The dataset resource to train the model.\n",
+        "- `target_column`: The column in the dataset that contains the values the model is trying to forecast.\n",
         "- `time_column`: Time-series column for the forecast model.\n",
         "- `time_series_identifier_column`: ID column for the time-series column.\n",
+        "- `available_at_forecast_columns`: List of columns that are available at the time of forecasting.\n",
+        "- `unavailable_at_forecast_columns`: List of columns that aren't available at the time of forecasting.\n",
+        "- `time_series_attribute_columns`: Columns that contain attributes or metadata related to the time series data, such as \"city,\" \"zip_code,\" and \"county\" in this example. These attributes can help the model understand the context of the time series.\n",
+        "- `forecast_horizon`: It determines how far into the future you want to predict, representing the number of time steps ahead for which the model generates predictions.\n",
+        "- `context_window`: The number of historical time steps the model uses as context for making predictions. A context window of 30 means the model uses data from the past 30 time steps to forecast future values.\n",
+        "- `data_granularity_unit`: The unit of time used for granularity in the data, such as \"day\" or \"hour.\" This specifies the time interval between data points.\n",
+        "- `data_granularity_count`: The count of the granularity unit. For example, a data_granularity_count of 1 with a `data_granularity_unit` of \"day\" means each data point represents one day.\n",
+        "- `weight_column`: parameter allows you to assign different weights to different data points in your training set.\n",
```

Comment:
add "This", change "allows" to "lets", and delete "to" : "This parameter lets you assign different weights..."